app/Stores.tsx

```diff
@@ -1,9 +1,7 @@
 import { ReactElement, useState } from "react";
-import { PageHeader } from "./ui/PageHeader";
 import { Wrapper } from "./ui/Wrapper";
 import { ViewStores } from "./ViewStores";
 import { CreateStore } from "./CreateStores";
-import { UpdateStore } from "./UpdateStores";
 
 enum StoresPage {
   View,
```

Comment:
I suppose `Update` is of no use in this enum now?

Reply:
I think you were looking at the first commit because this isn't in my version

Reply:
Don't think so, you just haven't changed this code at all, but `Update` should be removed

Reply:
OK BRO